### PR TITLE
gitignore: add .venv

### PR DESCRIPTION
**Pull request by GuillaumeCisco**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 docker-compose.yml
 docker-compose-dynamic.yaml
 docker-compose-dynamic-add.yaml
+**/.venv
```
Comment (ClementGautier) [inline .gitignore]: `**/` seems superfluous here, I suggest we use the recommended setup for python repository: https://github.com/github/gitignore/blob/master/Python.gitignore#L102 
```suggestion
.venv
```

**Comment by Kelvin-M** [comment]
@GuillaumeCisco Can you update your PR as suggested by Clément ? 
We will be able to merge it quickly as it is a minor modification ! 🚀

**Comment by GuillaumeCisco** [comment]
This is @AurelienGasser PR, I've just transferred it to the new repo.

**Comment by Kelvin-M** [comment]
Okay, sorry for the notification :)
Waiting for Aurelien !

**Comment by Kelvin-M** [comment]
Updated along Clement note !

**Pull request merged**